src/ServiceControl.Config/UI/Shell/ShellViewModel.cs

```diff
@@ -131,6 +143,28 @@ void LoadAppVersion()
             }
         }
 
+        private async Task CheckForUpdates()
+        {
+            // Get the lates upgradble version based on the current version
+            // get the json version file from https://s3.us-east-1.amazonaws.com/platformupdate.particular.net/servicecontrol.txt
+
+            var shortAppVersion = AppVersion.Substring(0, 6);
```

Comment:
This only works for the current version `4.12.1`. It won't work for future versions like `5.0.0`.

```suggestion
            var shortAppVersion = AppVersion.Split('-').First();
```